File: PY/Program/Sumisang.py
class Sumisang:
    def __init__(self, code="", name="", price=0):
        # Private attributes
        self.__code = code
        self.__name = name
        self.__price = price

    # getter dan setter untuk code
    def get_code(self):
        return self.__code

    def set_code(self, code):
        self.__code = code

    # getter dan setter untuk name
    def get_name(self):
        return self.__name
    
    def set_name(self, name):
        self.__name = name

    # getter dan setter untuk price
    def get_price(self):
        return self.__price

    def set_price(self, price):
        self.__price = price

File: PHP/Program/Sumisang.php
<?php
class Sumisang {
    private $code;
    private $name;
    private $price;
    private $photo;  // Menambahkan atribut untuk foto

    // Constructor
    public function __construct($code = "", $name = "", $price = 0, $photo = "") {
        $this->code = $code;
        $this->name = $name;
        $this->price = $price;
        $this->photo = $photo;  // Menyimpan foto
    }

    // Setters and Getters
    public function setCode($code) {
        $this->code = $code;
    }

    public function getCode() {
        return $this->code;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getName() {
        return $this->name;
    }

    public function setPrice($price) {
        $this->price = $price;
    }

    public function getPrice() {
        return $this->price;
    }

    // Setter dan Getter untuk foto
    public function setPhoto($photo) {
        $this->photo = $photo;
    }

    public function getPhoto() {
        return $this->photo;
    }
}
?>
